src/cls/_ZPM/PackageManager/Client/REST/PackageManagerClient.cls

```diff
@@ -128,14 +128,22 @@ Method SendAnalyticsToGoogle(action As %String, data As %DynamicObject)
     Set TrackingID = ##class(%ZPM.PackageManager.Client.Settings).GetAnalyticsTrackingId()
     Quit:(TrackingID = "") 0
     Set tRequest = ##class(%Net.HttpRequest).%New()
+    Set tRequest.FollowRedirect = 1
     Set tRequest.Server = "www.google-analytics.com"
     Set tRequest.SSLConfiguration = ..GetSSLConfiguration(tRequest.Server)
+
+    Set moduleName = data.name
+    Set status = $case($$$ISOK(data.success), 1: "Success", : "Failure")
+    Set version = data.version
+    Set irisPlatform = $system.Version.GetPlatform()
+    Set irisVersion = $system.Version.GetNumber()
+    Set irisProduct = $system.Version.GetProduct()
     Set requestString = "v=1&tid="_TrackingID
         _"&cid="_##class(%SYS.System).InstanceGUID()
-        _"&ds=web&an="_data.name_"&av="_data.version
-        _"&t=event&aiid="_$ZCONVERT(data."system_version", "O", "URL")_"&ec="_$ZCONVERT(action, "O", "URL")_"&ea="
-        _$case($$$ISOK(data.success), 1: "Success", : "Failure")_"&el="
-        _$ZCONVERT(data."error_msg", "O", "URL")
+        _"&ds=web"
+        _"&t=event"
+        _"&ec="_$ZCONVERT(moduleName, "O", "URL")_"&ea="_$ZCONVERT(status, "O", "URL")
+        _"&el="_$ZCONVERT(version_", "_irisProduct_", "_irisPlatform_", "_irisVersion, "O", "URL")
```

Comment:
Please replace this long string with a bunch of [SetParam](https://docs.intersystems.com/irislatest/csp/documatic/%25CSP.Documatic.cls?PAGE=CLASS&LIBRARY=%25SYS&CLASSNAME=%25Net.HttpRequest#SetParam)

Reply:
done